### PR TITLE
(PS-1934) Install msodbc driver

### PR DESCRIPTION
**Pull request by JakubMatejka**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,14 +3,19 @@ ENV PYTHONIOENCODING utf-8
...
         python-scipy \
         python-matplotlib \
         ipython \
+        msodbcsql17 \
+        mssql-tools \
         python-pandas \
         unixodbc-dev \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
+    && echo 'export PATH="$PATH:/opt/mssql-tools/bin"' >> ~/.bashrc
```
Comment (pivnicek) [inline Dockerfile:18]: Is this going to be valid for the sandbox user?  maybe we should set it globally?

**Review by pivnicek — COMMENTED** [review]
It built fine locally for me, but I'm worried that the PATH may not be set for the NB_USER

File: Dockerfile (hunk 1 above)
Comment (JakubMatejka) [reply Dockerfile:18]: 🤔 ah you mean like this? https://github.com/keboola/db-extractor-mssql/blob/master/Dockerfile#L47

Comment (pivnicek) [reply Dockerfile:18]: yeah, I think so.

Comment (pivnicek) [reply Dockerfile:18]: Because the jupyterlab is run under NB_USER rather than root so I'm afraid it won't load its shell from this bashrc

Comment (JakubMatejka) [reply Dockerfile:18]: Right, I understand now. 👍

**Review by pivnicek — APPROVED** [review]
supr

**Pull request merged**